alibi_detect/utils/pydantic_v1/tests/test_import_pydantic_v1.py

```diff
@@ -0,0 +1,6 @@
+def test_import():
+    from alibi_detect.utils.pydantic_v1 import BaseModel
```

Comment:
@jesse-c this is only testing one version of pydantic being installed (probably v2) as far as I can tell. We probably should have a way to add a test for the other version of pydantic being installed to really make sure that the shim is working as expected?

Reply:
Good point! I can do that. I took a quick look at how they do it with Ray [1] and it makes sense for a similar approach. I'll put something together to test out my idea.

[1] https://github.com/ray-project/ray/pull/40451

Reply:
I've created https://github.com/SeldonIO/alibi-detect/pull/890.